e2e/process-services-cloud/apps-section-cloud.e2e.ts

```diff
@@ -28,13 +29,33 @@ describe('Applications list', () => {
     const settingsPage = new SettingsPage();
     const appListCloudPage = new AppListCloudPage();
     const simpleApp = resources.ACTIVITI7_APPS.SIMPLE_APP.name;
+    let identityService: IdentityService;
+    let groupIdentityService: GroupIdentityService;
+    let rolesService: RolesService;
+    let testUser, apsUserRoleId, groupInfo;
+    const apiService = new ApiService(browser.params.config.oauth2.clientId, browser.params.config.bpmHost, browser.params.config.oauth2.host, 'BPM');
 
     beforeAll(async (done) => {
+        await apiService.login(browser.params.identityAdmin.email, browser.params.identityAdmin.password);
+        identityService = new IdentityService(apiService);
+        groupIdentityService = new GroupIdentityService(apiService);
+        rolesService = new RolesService(apiService);
+        testUser = await identityService.createIdentityUser();
+        apsUserRoleId = await rolesService.getRoleIdByRoleName(CONSTANTS.ROLES.APS_USER);
+        await identityService.assignRole(testUser.idIdentityService, apsUserRoleId, CONSTANTS.ROLES.APS_USER);
+        groupInfo = await groupIdentityService.getGroupInfoByGroupName('hr');
+        await identityService.addUserToGroup(testUser.idIdentityService, groupInfo.id);
         await settingsPage.setProviderBpmSso(
             browser.params.config.bpmHost,
             browser.params.config.oauth2.host,
             browser.params.config.identityHost);
-        loginSSOPage.loginSSOIdentityService(browser.params.identityUser.email, browser.params.identityUser.password);
+        loginSSOPage.loginSSOIdentityService(testUser.email, testUser.password);
+        done();
+    });
+
+    afterAll(async(done) => {
+        await apiService.login(browser.params.identityAdmin.email, browser.params.identityAdmin.password);
```

Comment:
Make sure the identity admin is exposed as travis env variable

Reply:
Can you pls expand as to what file are you referring to here pls?

Reply:
On the travis setting page
https://travis-ci.org/Alfresco/alfresco-ng2-components/settings
We should have an ENV variable that should be passed in the travis.yml file

Reply:
it's been handled.